### PR TITLE
Prepare environment to support guzzle 5 and 6

### PR DESCRIPTION
**Pull request by gianarb**
Fixed #40
@thecodeassassin

### CODE REVIEW

**Comment by gianarb** [comment]
Hello @tuupola can you try this remote please?
Thanks

**Comment by tuupola** [comment]
I do not know how to install it. I guess composer get confused when branch name contains `/` character. However looking at the code this change should fix my problem.

```bash
$ composer require influxdb/influxdb-php:dev-feature/guzzle5-6
./composer.json has been updated
Loading composer repositories with package information
Updating dependencies (including require-dev)
Your requirements could not be resolved to an installable set of packages. 

  Problem 1
    - The requested package influxdb/influxdb-php dev-feature/guzzle5-6 could not be found.
```

**Comment by thecodeassassin** [comment]
@gianarb i don't think having your personal access token in a public repo is a good idea since it can be used to access your account. I suggest changing the composer install to use --no-interaction and/or --prefer-source, this way composer will fallback to clone if it reaches the github api limit (instead of downloading the achive).

**Comment by gianarb** [comment]
@tuupola @thecodeassassin I updated our build in order to test all our dependencies but Guzzle 5 is not supported because we use `MockHandler` for all our tests, this class is provided only in Guzzle 6, for guzzle file exists a similar class I think
http://docs.guzzlephp.org/en/5.3/handlers.html

but in practice at the moment this library is not ready to support both version

**Comment by tuupola** [comment]
Yeah, the tests could be restructured so that drivers are tested separately and the client is not tested with specific Guzzle driver but with NullDriver or something similar instead. I can take a look on that bit later.

**Comment by gianarb** [comment]
Thanks @tuupola !

**Comment by thecodeassassin** [comment]
@gianarb is this PR still relevant?

File: .travis.yml [hunk 1]
```diff
@@ -6,9 +6,41 @@ php:
...
+      env:
+        - GUZZLE="~5.0"
+        - SYMFONY_EVENT_DISPATCHER="~2.0"
+    - php: 5.6
+      env:
+        - GUZZLE="~6.0"
+        - SYMFONY_EVENT_DISPATCHER="~2.0"
+    - php: 7.0
+      env:
+        - GUZZLE="~6.0"
```
Comment (virgofx) [inline .travis.yml]: Should this be Guzzle ~5.0?

**Pull request closed without merging**